auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/AuctionApplication.java

```diff
@@ -10,4 +13,9 @@ public static void main(String[] args) {
 		SpringApplication.run(AuctionApplication.class, args);
 	}
 
+	@Bean
+	PasswordEncoder passwordEncoder() {
+		return new BCryptPasswordEncoder();
+	}
```

Comment:
Not sure if this should be placed in this class. If we need this bean, I suggest creating new configuration class and putting this there.

Reply:
Sure, will do.